lib/relations_sql/expression.py

```diff
@@ -160,18 +160,18 @@ def generate(self, **kwargs):
         self.args = []
 
 
-class SCHEMA(NAME):
+class SCHEMANAME(NAME):
     """
     For schemas
     """
 
 
-class TABLE(SCHEMA):
+class TABLENAME(SCHEMANAME):
```

Comment:
Same question [as above.](https://github.com/gaf3/python-relations-sql/pull/10/files#r707795137)

Reply:
Because it's really the SQL for for table name, not a table itself. When there was no DDL (which has TABLE now), the shorthand was fine. Now that there is a TABLE DDL, we don't want name collisions.